IntegrationTests.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine3.8
```

Comment:
I think it's better to stay currently on the same python version as in our release image which is python2

Reply:
That does not really matter to be honest. This tests are not working with any of the code of the actual project and as it is new code I would really stay with python3 here. 

Python2 is deprecated since forever.

Reply:
Ah, true!